db/migrate/20240108184340_create_posts.rb

```diff
@@ -3,10 +3,9 @@ def change
     create_table :posts do |t|
       t.string :title
       t.text :text
-      t.integer :likes_counter, default: 0
-      t.integer :comments_counter, default: 0
-      t.references :author, foreign_key: { to_table: :users }
-
+      t.integer :likes_counter
+      t.integer :comments_counter
+      t.references :author, foreign_key: true
```

Comment:
- [ ] 👏 Excellent work on addressing the previous reviewer's request. However, you have updated the migration file after the previous reviewer's request. The current version is causing an error when running `rails db: migrate`, because table with named "author" is not present. It appears that the previous version was correct. I recommend reverting your migration file to the previous version that worked correctly. This will ensure that the database migration process runs smoothly without encountering errors.

![image](https://github.com/evansnyamekye/Blog-application/assets/55840999/693b7d34-cb84-4455-b19c-e77b99370c94)
Keep up the good work, and feel free to reach out if you have any questions or need further assistance! 🚀🛠️

Reply:
Hi @shahadat3669 
![su](https://github.com/evansnyamekye/Blog-application/assets/60971417/41870e29-618b-4a82-8ef5-fecf028d5d0d)

I migrated it and it was successful. could you please send a short video of the entire codebase of the problem so I resolved it

Reply:
@evansnyamekye, I understand. I suggest dropping your database, creating it again, and then running migrations. You will get the same errors.